node/slack-command-bot/src/utils.js

```diff
@@ -0,0 +1,50 @@
+import crypto from 'crypto';
+
+/**
+ * Throws an error if any of the keys are missing from the object
+ * @param {*} obj
+ * @param {string[]} keys
+ * @throws {Error}
+ */
+export function throwIfMissing(obj, keys) {
+  const missing = [];
+  for (let key of keys) {
+    if (!(key in obj) || !obj[key]) {
+      missing.push(key);
+    }
+  }
+  if (missing.length > 0) {
+    throw new Error(`Missing required fields: ${missing.join(', ')}`);
+  }
+}
+
+/**
+ * Throws an error if any of the keys are missing from the object
+ * @param {string} timestamp
+ * @param {string} bodyRaw
+ * @param {string} signature
+ * @throws {Error}
+ */
+export function throwIfRequestNotVerified(timestamp, bodyRaw, signature){
```

Comment:
Can this function just accept req as the argument and access the headers inside?